tests/performance/helper/helper.go

```diff
@@ -67,3 +116,70 @@ func RedefinePodWithSharedContainer(pod *corev1.Pod, containerIndex int) {
 		}
 	}
 }
+
+func ChangeSchedulingPolicy(pod *corev1.Pod, command string) (outStr, errStr string, err error) {
+	outStr, errStr, err = ExecCommandContainer(pod, command)
+	if err != nil {
+		return "", "", fmt.Errorf("cannot execute command: \" %s \"  on %s err:%w", command, pod.Spec.Containers[0].Name, err)
+	}
+
+	return outStr, errStr, err
+}
+
+// ExecCommand runs command in the pod and returns buffer output.
+func ExecCommandContainer(
+	pod *corev1.Pod, command string) (stdout, stderr string, err error) {
+	commandStr := []string{"sh", "-c", command}
+
+	var buffOut bytes.Buffer
+
```

Comment:
No need for the extra space.

Reply:
These are added because go-linter failed in the CI pipeline.